CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Added
+- Added testing via CircleCI
+
+### Deprecated
+- Deprecated animation plugin, consider using Tailwind's animation utilities (1.6+) instead
```

Comment:
@gregkohn I had time to revisit this PR, restoring the animation plugin and marking it for deprecation instead. Also, I need to rebase and update the changelog, since testing has actually been released.